### DISCUSSION TITLE
where the heck is config.php?

### DISCUSSION THREAD

**Discussion by bobloadmire** [nextcloud/all-in-one#2420] [Questions] [answered]
I'm trying to run nextcloud in docker desktop on windows behind a cloudflare tunnel, so I need to disable the domainchecker by adding "SKIP_DOMAIN_VALIDATION=true" but I cannot for the life of me find it to edit it. I also tried adding SKIP_DOMAIN_VALIDATION=true as an environmental variable in Portainer, but that completely breaks the container and it won't boot.

any advice would be great. I'm following this guide: https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

specifically the cloudflare tunnel section.

**Comment by Zoey2936** [comment, marked as answer]
did you used the compose file?

**Comment by bobloadmire** [comment]
I'm searching the file system for it
